app/css/cohortbuilder.css

```diff
@@ -107,6 +107,28 @@ li>a:hover {
 li>a:focus {
     color: gold;
 }
+.table thead {
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
+    position: sticky;
+    top: 0;
+    background-color: #FFF;
+}
+.table {
+    table-layout: fixed;
+    width: 100%;
+}
+
+th, td {
+    width: 180px;
+    word-wrap:break-word;
+}
+
+.index-size {
```

Comment:
I have opted to use table-header due to the convention followed in [previous css files](https://github.com/openmrs/openmrs-owa-cohortbuilder/blob/master/app/css/cohortbuilder.css) except for a few lines that might need a rework so they are all consistent. CamelCasing somehow makes it harder to read the css.